amethyst_network/src/resources/net_event.rs

```diff
@@ -4,9 +4,26 @@ use resources::connection::*;
 
 ///The basic network events shipped with amethyst
 ///You can add more event by making a new enum and having this one has a variant of yours.
+/// #[derive(Debug,Clone,PartialEq,Serialize,Deserialize)]
```

Comment:
Should go into doc example (\```rust at the beginning and ``` at the end)

Reply:
fixed